dts/bindings/usb/st,stm32-otgfs.yaml

```diff
@@ -31,5 +31,16 @@ properties:
     type: phandle
     description: PHY provider specifier
 
+  phy-type:
+    type: int
+    description: >
+      Type of PHY (Physical Layer) interface used by the OTG_FS/OTG_HS interface.
+
+      The general features of the OTG_FS/OTG_HS interface are as follows:
+
+        - 1: Uses the ULPI interface for an external high-speed PHY for OTG_HS.
+        - 2: Uses the on-chip full-speed PHY for OTG_FS.
+        - 3: Uses the on-chip full-speed PHY for OTG_HS.
```

Comment:
Let's replace by a string enum. See https://github.com/zephyrproject-rtos/zephyr/pull/82955 for instance.